awx/main/models/workflow.py

```diff
@@ -79,6 +79,12 @@ class Meta:
         symmetrical=False,
         related_name='%(class)ss_always',
     )
+    all_parents_must_converge = models.BooleanField(
+        'self',
```

Comment:
What is "self" indicating here?

Reply:
the migration interpreted that as `verbose_name`. I don't like that, let's get rid of it.

Reply:
I'll remove and push, that makes good sense